content/en/blog/2025/upgrade_from_3.5_to_3.6_issue_followup.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Follow Up - Preventing Upgrade Failures from etcd v3.5 to v3.6
+author:  "[Benjamin Wang](https://github.com/ahrtr) VMware by Broadcom, [Josh Berkus](https://github.com/jberkus) Red Hat"
+date: 2025-10-20
+draft: false
+---
+
+We have identified and fixed an additional scenario that may cause upgrade failures when
+moving from etcd v3.5 to v3.6. This post contains details, the fix, and additional workarounds.
+Please refer to issue [20793][] to get detailed technical information.
```

Comment:
I think we should close the issue https://github.com/etcd-io/etcd/issues/20793 to assure users that it's fixed.

Reply:
Sure! I meant to say before we publish this blog. So we are on same page.

Reply:
Thanks @ahrtr !!